### PR TITLE
assignments 2-4

### PR DESCRIPTION
**Pull request by sw-upm**
sw-upm

### CODE REVIEW

File: Assignment4/sw-upm-21A309/odkg_assignment_4_task06.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+# -*- coding: utf-8 -*-
+"""Task06.ipynb
+Automatically generated by Colaboratory.
+Original file is located at
+    https://colab.research.google.com/drive/1hSv61ejXESrp34CUz1PqZCsoA43tzc8y
+**Task 06: Modifying RDF(s)**
+"""
+
+!pip install rdflib
```
Comment (dgarijo) [inline Assignment4/sw-upm-21A309/odkg_assignment_4_task06.py:18]: this won't work in Python, as it's notebook magic

File: Assignment4/sw-upm-21A309/odkg_assignment_4_task07.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        print(sub)
+
+print("List all individuals of \"Person\" via SPARQL:")
+q = prepareQuery('''
+  SELECT ?Subject
+  WHERE { 
+    {?Subject rdf:type ns:Person} 
+    UNION
+    {
+     ?Subclass rdfs:subClassOf ns:Person .
```
Comment (dgarijo) [inline Assignment4/sw-upm-21A309/odkg_assignment_4_task07.py:68]: use * to get all subclasses of Person, not direct subclasses only

**Review by dgarijo — CHANGES_REQUESTED** [review]
Small changes requested. @ocorcho there is a review pending from Assignment 2 and 3

**Comment by dgarijo** [comment]
@ocorcho please review assignments 2-3. Assignment 4 is ok

File: Assignment4/sw-upm-21A309/odkg_assignment_4_task07.py (hunk 2 above)
Comment (sw-upm) [reply Assignment4/sw-upm-21A309/odkg_assignment_4_task07.py]: I changed this correspondingly in the latest commit

File: Assignment4/sw-upm-21A309/odkg_assignment_4_task07.py [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+print("List all individuals of \"Person\" and all their properties including their classvia SPARQL")
+q = prepareQuery('''
+  SELECT ?Subject ?Pre ?Obj
+  WHERE {
+    {?Subject rdf:type ns:Person .
+     ?Subject ?Pre ?Obj
+    }
+    UNION 
+    {
+     ?Subclass rdfs:subClassOf ns:Person .
```
[the remark this answers is not in the document]
Comment (sw-upm) [reply Assignment4/sw-upm-21A309/odkg_assignment_4_task07.py]: I changed this correspondingly in the latest commit

**Pull request merged**